src/suites/cts/validation/setVertexBuffers.spec.ts

```diff
@@ -0,0 +1,184 @@
+export const description = `
+setVertexBuffers validation tests.
+`;
+
+import { TestGroup } from '../../../framework/index.js';
+
+import { ValidationTest } from './validation_test.js';
+
+class F extends ValidationTest {
+  async init(): Promise<void> {
+    await Promise.all([super.init(), this.initGLSL()]);
+  }
+
+  getVertexBuffer(): GPUBuffer {
+    return this.device.createBuffer({
+      size: 256,
+      usage: GPUBufferUsage.VERTEX,
+    });
+  }
+
+  createRenderPipeline(bufferCount: number): GPURenderPipeline {
+    const descriptor: GPURenderPipelineDescriptor = {
+      vertexStage: this.getVertexStage(bufferCount),
+      fragmentStage: this.getFragmentStage(),
+      layout: this.getPipelineLayout(),
+      primitiveTopology: 'triangle-list',
+      colorStates: [{ format: 'rgba8unorm' }],
+      vertexInput: {
+        vertexBuffers: [
+          {
+            stride: 3 * 4,
+            attributeSet: [],
```

Comment:
if we had a helper in a common file somewhere:

```js
function range<T>(n: number, fn: (i: number): T) { return [...Array(n).keys()].map(fn); }
```

then we could use it naturally here:

```js
attributeSet: range(bufferCount, i => { shaderLocation: i, format: 'float3' }),
```

Reply:
Done

Reply:
Note that I had to use `new Array` because `Calls to 'Array' are not allowed. tsstyle#array-constructor (ban)`